CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# Developer guide
+
+## CI
+
+Checks are automatically run on certain pushed commits for testing and linting
+purposes. Some are defined by [.github/workflows/ci.yaml][] while others are
+configured outside of this repository.
```

Comment:
_non-blocking_

To prevent text here from getting out of sync with the list of event triggers in the CI workflow, I would just point to the workflow itself. 

```suggestion
Tests are run through GitHub Actions when triggered by events as defined by [.github/workflows/ci.yaml][] 
```